compose/cli/main.py

```diff
@@ -1392,10 +1398,13 @@ def log_printer_from_project(
         log_args=log_args)
 
 
-def filter_containers_to_service_names(containers, service_names):
+def filter_attached_containers(containers, service_names, attach_dependencies):
```

Comment:
```suggestion
def filter_attached_containers(containers, service_names, attach_dependencies=False):
```
By using a default value we can keep the same usage.
With this you can simplify the calls in the tests that are not related to the PR.

Reply:
Good suggestion - updated 👍 .